src/main/CommandLine.cpp

```diff
@@ -1820,6 +1822,45 @@ runGenFuzz(CommandLineArgs const& args)
             return 0;
         });
 }
+
+int
+runApplyLoad(CommandLineArgs const& args)
+{
+    CommandLine::ConfigOption configOption;
+
+    return runWithHelp(args, {configurationParser(configOption)}, [&] {
+        auto config = configOption.getConfig();
+        config.RUN_STANDALONE = true;
+
+        VirtualClock clock(VirtualClock::REAL_TIME);
+        int result;
+        auto appPtr = Application::create(clock, config);
+
+        auto& app = *appPtr;
+        {
+            auto& lm = app.getLedgerManager();
+            app.start();
+            ApplyLoad al(app);
+
+            auto& ledgerClose =
```

Comment:
I don't have too strong opinion here, but it could be more convenient to encapsulate the metric semantics inside `benchmark` function and then return a struct with interesting metrics. Or maybe just output everything there. Though it depends on how we're going to call this, so maybe this is a fine place as well; up to you.